### ISSUE TITLE
How to run classification_flow.py, ERROR: unrecognized arguments

### ISSUE THREAD

**Issue by Li-Whasaka** [NVIDIA/TensorRT#3386]
I iput this command : **python classification_flow.py -d zxzx/TensorRT-release-8.5/tools/pytorch-quantization/examples/mnist  --ckpt-path /home/lwx/Desktop/lwx/zxzx/TensorRT-release-8.5/tools/pytorch-quantization/examples/torchvision/resnet50-19c8e357.pth**

and it showed:

TensorRT-release-8.5/tools/pytorch-quantization/examples/mnist  --ckpt-path /home/lwx/Desktop/lwx/zxzx/TensorRT-release-8.5/tools/pytorch-quantizat
ion/examples/torchvision/resnet50-19c8e357.pth 
Classification quantization flow script
Namespace(batch_size_onnx=1, batch_size_test=128, batch_size_train=128, calibrator='max', ckpt_path='/home/lwx/Desktop/lwx/zxzx/TensorRT-release-8.5/tools/pytorch-quantization/examples/torchvision/resnet50-19c8e357.pth', ckpt_url='', data_dir='zxzx/TensorRT-release-8.5/tools/pytorch-quantization/examples/mnist', disable_pcq=False, evaluate_onnx=False, model_name='resnet50', num_calib_batch=4, num_finetune_epochs=0, out_dir='/tmp', percentile=[99.9, 99.99, 99.999, 99.9999], pretrained=False, print_freq=20, seed=12345, sensitivity=False, threshold=-1.0)
usage: classification_flow.py [-h] [--data-path DATA_PATH] [--model MODEL] [--device DEVICE] [-b BATCH_SIZE] [--epochs N] [-j N] [--opt OPT]
                              [--lr LR] [--momentum M] [--wd W] [--norm-weight-decay NORM_WEIGHT_DECAY] [--bias-weight-decay BIAS_WEIGHT_DECAY]
                              [--transformer-embedding-decay TRANSFORMER_EMBEDDING_DECAY] [--label-smoothing LABEL_SMOOTHING]
                              [--mixup-alpha MIXUP_ALPHA] [--cutmix-alpha CUTMIX_ALPHA] [--lr-scheduler LR_SCHEDULER]
                              [--lr-warmup-epochs LR_WARMUP_EPOCHS] [--lr-warmup-method LR_WARMUP_METHOD] [--lr-warmup-decay LR_WARMUP_DECAY]
                              [--lr-step-size LR_STEP_SIZE] [--lr-gamma LR_GAMMA] [--lr-min LR_MIN] [--print-freq PRINT_FREQ]
                              [--output-dir OUTPUT_DIR] [--resume RESUME] [--start-epoch N] [--cache-dataset] [--sync-bn] [--test-only]
                              [--auto-augment AUTO_AUGMENT] [--ra-magnitude RA_MAGNITUDE] [--augmix-severity AUGMIX_SEVERITY]
                              [--random-erase RANDOM_ERASE] [--amp] [--world-size WORLD_SIZE] [--dist-url DIST_URL] [--model-ema]
                              [--model-ema-steps MODEL_EMA_STEPS] [--model-ema-decay MODEL_EMA_DECAY] [--use-deterministic-algorithms]
                              [--interpolation INTERPOLATION] [--val-resize-size VAL_RESIZE_SIZE] [--val-crop-size VAL_CROP_SIZE]
                              [--train-crop-size TRAIN_CROP_SIZE] [--clip-grad-norm CLIP_GRAD_NORM] [--ra-sampler] [--ra-reps RA_REPS]
                              [--weights WEIGHTS] [--backend BACKEND] [--use-v2]
classification_flow.py: error: unrecognized arguments: -d zxzx/TensorRT-release-8.5/tools/pytorch-quantization/examples/mnist --ckpt-path /home/lwx/Desktop/lwx/zxzx/TensorRT-release-8.5/tools/pytorch-quantization/examples/torchvision/resnet50-19c8e357.pth

SO, Could someone tell me the correct COMMAND to RUN this program here?

**Comment by zerollzeng** [comment]
Are you trying to specify `--data-path DATA_PATH`?

**Comment by Li-Whasaka** [comment]
> Are you trying to specify `--data-path DATA_PATH`?

sure，showed same error.

**Comment by Li-Whasaka** [comment]
sure，showed same error.

**Comment by zerollzeng** [comment]
Could you please try the latest code? e.g. 8.6 branch.

**Comment by ttyio** [comment]
closing since no activity for more than 3 weeks, thanks!
